app/views/comments/unreviewed.html.erb

```diff
@@ -3,10 +3,12 @@
 <!--/descriptions-->
 
 <!--Subnavigation, sorting and actions-->
-<ul class="navigation actions">
-  <% path = @commentable.is_a?(AdminPost) ? review_all_admin_post_comments_path(@commentable) : review_all_work_comments_path(@commentable) %>
-  <li><%= button_to(t(".approve_all"), path, method: :put) %></li>
-</ul>
+<% if @comments[0] && can_review_comment?(@comments[0]) %>
```

Comment:
I think `@comments.first` would be the more typical way to get the first comment in the array. 

That being said, if we have to deal with the possibilities of empty arrays here, it's probably tidier to define a new helper method that takes the commentable, e.g. 

```
def can_review_all_comments?(commentable)
  commentable.is_a?(AdminPost) || is_author_of?(commentable)
end
```